conf/messages.en

```diff
@@ -197,19 +197,11 @@ amountsAlreadyInCode.paragraph1.multiple = Your tax-free Personal Allowance alre
 amountsAlreadyInCode.paragraph2 = Your tax code was changed the first time that you claimed this expense, and will stay the same until you tell us that your professional subscriptions circumstances change.
 amountsAlreadyInCode.label.single = Do you want to change this amount?
 amountsAlreadyInCode.label.multiple = Do you want to change any of these amounts?
-amountsAlreadyInCode.checkYourAnswersLabel = Are the amounts already in your tax code correct? {0}
+amountsAlreadyInCode.checkYourAnswersLabel = Do you want to change this amount? {0}
```

Comment:
this was added, due to it being missed in a previous ticket to update this page.